cloudcomposerdiff/README.md

```diff
@@ -0,0 +1,184 @@
+# A python command line tool to diff two Cloud composer environments.
+
+## Running this code
+
+### Clone repository
+
+Clone the code from GitHub to your local machine
+
+### Install locally with pip
+
+```
+pip install cloudcomposerdiff
+```
+
+### Authorizing the application using gcloud
+
+Install [gcloud](https://cloud.google.com/sdk/gcloud)
+
+Select a user which has access to both composer environments to compare.
+
+Wit this user account, initialize gcloud:
+
+```
+gcloud init
+```
+
+Next, authorize the application to access cloud composer environments:
+
+```
+gcloud auth application-default login
+```
+
+### Use the appplication
```

Comment:
This section needs more info. What should the user expect to happen when they run this command? (how long will it take? minutes? hours? days? What happens when it runs?) Tell them here

Reply:
Thank you for the feedback. I have added several videos (in the form of GIFs) to the README.   These should clearly explain to users what will happen when they run the tool.  Please let me know if the GIFs are not clear.